clients/ui/frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18 as build-stage
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
```

Comment:
ok, a couple of annotations, this is just working cause you are copying your local `node_modules`, but that's a big nono, we should copy ONLY the files that are relevant to the project, we can either have `dockerignores` or something similar but this can cause some issues and won't work in a computer without a installation of the packages.

See the dockerignore we have here: https://github.com/opendatahub-io/odh-dashboard/blob/main/.dockerignore

And that we copy and do a ci before building: https://github.com/opendatahub-io/odh-dashboard/blob/main/Dockerfile#L15-L22

Reply:
but this is only copied across during the build stage? Those artefacts are not copied across to the runtime container? Unless I'm misunderstanding something.